lib/kaguya/core.ex

```diff
@@ -15,6 +15,7 @@ defmodule Kaguya.Core do
   defp password, do: Application.get_env(:kaguya, :password)
   defp use_ssl, do: Application.get_env(:kaguya, :use_ssl)
   defp reconnect_interval, do: Application.get_env(:kaguya, :reconnect_interval)
+  defp server_timeout, do: Application.get_env(:kaguya, :server_timeout)
```

Comment:
By default it should turned off (i.e. it should return nil when not specified) The intention is that this server_timeout is optional and when omitted it is disabled.
But if you would like to have it always then i could use default value as 1m